incubator/kafka/templates/service-brokers-external.yaml

```diff
@@ -8,6 +8,7 @@
     {{- $externalListenerPort := add $root.Values.external.firstListenerPort $i }}
     {{- $responsiblePod := printf "%s-%d" (printf "%s" $fullName) $i }}
     {{- $distinctPrefix := printf "%s-%d" $dnsPrefix $i }}
+    {{- $loadBalancerIP := index $root.Values.external.loadBalancerIP $i }}
```

Comment:
If not loadbalancer ips are set, it throws

```
Error: render error in "kafka/charts/kafka/templates/service-brokers-external.yaml": template: kafka/charts/kafka/templates/service-brokers-external.yaml:11:27: executing "kafka/charts/kafka/templates/service-brokers-external.yaml" at <index $root.Values.e...>: error calling index: index out of range: 0
```

Can we make the static ip optional ?

Reply:
Added PR #10816